skimage/future/detect_obj/cascade.pyx

```diff
@@ -0,0 +1,836 @@
+        provided image, carries out classification and gives a binary result.
+
+        Parameters
+        ----------
+        int_img : float[:, ::1]
+            Memory-view to integral image.
+        row : Py_ssize_t
+            Row coordinate of the rectangle in the given image to classify.
+            Top left corner of window.
+        col : Py_ssize_t
+            Column coordinate of the rectangle in the given image to classify.
+            Top left corner of window.
+        scale : float
+            The scale by which the search window is multiplied.
+            After multiplication the result is rounded to the lowest integer.
+
+        Returns
+        -------
+        result : int
+            The binary output that takes only 0 or 1. Gives 1 if the classifier
+            detects the object in specified region and 0 otherwise.
+        """
+
+        cdef:
+            float stage_threshold
+            float stage_points
+            int lbp_code
+            int bit
+            Py_ssize_t stage_number
+            Py_ssize_t weak_classifier_number
+            Py_ssize_t feature_number
+            Py_ssize_t features_amount
+            Py_ssize_t stumps_amount
+            Py_ssize_t first_stump_idx
+            Py_ssize_t lut_idx
+            Py_ssize_t r, c, widht, height
+            cnp.uint32_t[::1] current_lut
+            Stage current_stage
+            MBLBPStump current_stump
+            MBLBP current_feature
+
+
+        for stage_number in range(self.stages_amount):
+
+            current_stage = self.stages[stage_number]
+            first_stump_idx = current_stage.first_idx
+            stage_points = 0
+
+            for weak_classifier_number in range(current_stage.amount):
+
+                current_stump = self.stumps[first_stump_idx + weak_classifier_number]
+
+                current_feature = self.features[current_stump.feature_id]
+
+                r = <Py_ssize_t>(current_feature.r * scale)
+                c = <Py_ssize_t>(current_feature.c * scale)
+                width = <Py_ssize_t>(current_feature.width * scale)
+                height = <Py_ssize_t>(current_feature.height * scale)
+
+
+                lbp_code = _multiblock_lbp(int_img,
+                                           row + r,
+                                           col + c,
+                                           width,
+                                           height)
+
+                lut_idx = current_stump.lut_idx
+
+                bit = (self.LUTs[lut_idx + (lbp_code >> 5)] >> (lbp_code & 31)) & 1
+
+                stage_points += current_stump.left if bit else current_stump.right
+
+            if stage_points < (current_stage.threshold - self.eps):
+
+                return False
+
+        return True
+
+    def _get_valid_scale_factors(self, min_size, max_size, scale_step):
+        """Get the valid scale multipliers for the original window size
```

Comment:
This short doc string needs to be one line only according to the docstring conventions.